docs/tutorials/agent_framework/RAG_with_conversational_flow_agent.md

```diff
@@ -0,0 +1,395 @@
+# Topic
+
+> Agent Framework is an experimental feature released in OpenSearch 2.12 and not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/ml-commons/issues/1161).
```

Comment:
OpenSearch 2.12 not released yet. If you want to try this feature now, you can test with the latest RC https://github.com/opensearch-project/opensearch-build/issues/4115#issuecomment-1939453371